Product/Production/Gateway/AdminGUIWebservices/src/main/java/gov/hhs/fha/nhinc/configadmin/ConfigAdmin.java

```diff
@@ -630,13 +643,21 @@ public SimpleCertificateResponseMessageType importToKeystore(ImportToKeystoreReq
         if (StringUtils.isBlank(alias) || null == request.getServerCert()) {
             return buildSimpleResponse(false, "Alias and Server Certificate are required.");
         }
+
+        PrivateKey privateKey = readPrivateKey(alias, getPasswordKeystore());
+        if (null == privateKey) {
+            return buildSimpleResponse(false, "Error no privatekey recorded for alias: " + alias);
+        }
+
+        tempKeystore = null;
+        if (!copyFile(FILE_JKS_GATEWAY, FILE_JKS_GATEWAY_NEW)) {
+            return buildSimpleResponse(false, "unable to make a copy of gateway-jks.");
```

Comment:
"Unable to copy keystore to temporary file" - the user might not have their file called "gateway", so lets generalize this error.